openshift/catalogd/manifests/11-rolebinding-openshift-config-catalogd-manager-rolebinding.yml

```diff
@@ -13,5 +13,5 @@ roleRef:
   name: catalogd-manager-role
 subjects:
   - kind: ServiceAccount
-    name: catalogd-controller-manager
```

Comment:
Why are we changing the name of the SA?

Reply:
good catch. If you look at PR changes, I'm not changing anything within config folder so it's interesting. I'll look into it

Reply:
Can you please reopen this one? 
It still changed. Did you check why?